### ISSUE TITLE
[Feature] Customizable keyboard shortcuts

### ISSUE THREAD

**Issue by mehturt** [traxium/tabtree#49]
I read today that Tab Groups (Panorama) is going to be removed from FF so I tried this tab with my Nightly. I see all the tabs on the left, which is good, but a few things don't work:
1. when typing part of the tab name (URL) in the search box, the tab is found (pink rectangle) but it's not focused - if it's scrolled out, it remains scrolled out of view
2. clicking on tab does not open this tab, but some other tab instead.. - this makes the extension unusable
Please let me know if you need more information, I'd love to use this extension.
Thanks.
m.

**Comment by jtojnar** [comment]
If you have multiple tab groups in panorama, the switching won’t work (#16).

**Comment by mehturt** [comment]
Yes I read that Tab Groups does not work with this addon, but I'm missing info on what I need to do to make this addon work? I have multiple tab groups right now, do I need to group all my tabs to one group and then enable this addon?
Thanks.

**Comment by jtojnar** [comment]
If I recall correctly, I moved my tabs to one group while having the add-on enabled and it worked just fine.

**Comment by mehturt** [comment]
Cool, I'll try that. Thanks.

**Comment by traxium** [comment]
It's good to know Firefox version number. And by default "Tab search jumps to the first match" option is disabled and it just highlights all matching tabs.

**Comment by mehturt** [comment]
The version may change every day, currently it's 45.0a1
https://nightly.mozilla.org/

**Comment by mehturt** [comment]
In my other profile, where I did not have tab groups, it works as expected. This issue can now be closed.
Btw. is there a keybinding to focus the search bar?

**Comment by traxium** [comment]
Ctrl+Alt+Shift+F focuses the search bar or if it's hidden brings it up and focuses.

**Comment by mehturt** [comment]
ah, that's too complex. is there a way to assign keybinding?

**Comment by traxium** [comment]
Not yet.

**Comment by roryokane** [comment]
[My comment in #144](https://github.com/traxium/tabtree/issues/144#issuecomment-255290131) may help with implementing this. It points out where key combinations are currently hard-coded except for the configurable F8 shortcut, and it describes two possible ways to load key combination descriptions from saved preferences so that you can check whether that key combination has just been pressed.
